<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-75MFPH4" failures="1" tests="1" name="testRunner.TestNGTestRunner" time="13.677" errors="0" timestamp="2023-02-01T12:33:33 PST" skipped="0">
  <testcase classname="testRunner.TestNGTestRunner" name="runScenario" time="13.677">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at stepDefinitions.HomePageSteps.validate_if_login_is_successfull(HomePageSteps.java:31)
at âœ½.validate if login is successfull(file:///E:/Java_workspace/CucumberFramework/./Features/Login.feature:6)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- testRunner.TestNGTestRunner -->
